pytorch_lightning/loops/fit_loop.py

```diff
@@ -97,6 +98,12 @@ def min_steps(self) -> int:
         """Returns the minimum numnber of steps to run"""
         return self.epoch_loop.min_steps
 
+    @min_steps.setter
+    def min_steps(self, value: int) -> None:
+        """Sets the minimum number of steps (forwards to epoch_loop)"""
+        # TODO(@awaelchli): This setter is required by debugging connector (fast dev run), should be avoided
+        self.epoch_loop.min_steps = value
+
```

Comment:
Why is this necessary now but not before?

Reply:
There is one setter for max_steps. I just added it for consistency.